recipe/meta.yaml

```diff
@@ -39,33 +39,36 @@ requirements:
     # llvm is needed for the headers
     - llvm-openmp              # [osx]
     - m2-patch  # [py==310 and win]
+    
 
   host:
     - python
     - pip
     - setuptools
     - wheel
-    - llvmlite 0.39.*
+    - llvmlite =0.40.0
```

Comment:
why is the '=' necessary?

Reply:
it could be either way, I dunno actually which one is prefered, anyone ?

Reply:
Generally we don't include '=' since it's redundant.

Reply:
done